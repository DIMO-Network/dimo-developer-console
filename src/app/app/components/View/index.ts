import View from './View';

export { View };

export default View;
