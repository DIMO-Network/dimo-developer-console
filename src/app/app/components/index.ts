export * from './OnboardingCard';
export * from './Onboarding';
