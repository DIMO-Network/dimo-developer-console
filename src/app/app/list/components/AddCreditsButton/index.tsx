import React from 'react';
import { PlusIcon } from '@heroicons/react/24/outline';
import { Button } from '@/components/Button';
import clsx from 'classnames';
import useOnboarding from '@/hooks/useOnboarding';

interface Props {
  className?: string;
}

const AddCreditsButton: React.FC<Props> = ({ className = 'dark with-icon' }) => {
  const { handleOpenBuyCreditsModal } = useOnboarding();
  return (
    <Button className={clsx(className, '!h-10')} onClick={handleOpenBuyCreditsModal}>
      <PlusIcon className="w-4 h-4" />
      Add credits
    </Button>
  );
};
export default AddCreditsButton;
