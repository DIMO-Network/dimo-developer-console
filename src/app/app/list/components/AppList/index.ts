export * from './AppList';
export { LicenseList } from '@/app/license/list/LicenseList';
