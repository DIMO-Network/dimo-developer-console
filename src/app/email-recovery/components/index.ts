export * from './EmailRecoveryForm';
export * from './CheckEmail';