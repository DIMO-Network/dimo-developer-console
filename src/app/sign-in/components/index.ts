export * from './OtpInputForm';
export * from './PasskeyLogin';
export * from './SignInMethodForm';
