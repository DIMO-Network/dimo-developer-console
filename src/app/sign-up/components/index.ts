export * from './BuildForForm';
export * from './CompanyInfoForm';
export * from './SignUpWith';
export * from './UserInfoForm';
