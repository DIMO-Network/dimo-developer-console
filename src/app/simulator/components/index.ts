export { View } from './View';
export { Header } from './Header';
export { SimulatedVehiclesSection } from './SimulatedVehiclesSection';
export { CreateSimulatedVehicleModal } from './CreateSimulatedVehicleModal';
