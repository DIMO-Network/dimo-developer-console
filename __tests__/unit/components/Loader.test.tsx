import { render } from '@testing-library/react';
import { Loader } from '@/components/Loader';

describe('Loader', () => {
  it('renders a loader component', () => {
    const { container } = render(<Loader isLoading />);

    expect(container).toMatchSnapshot();
  });
});
