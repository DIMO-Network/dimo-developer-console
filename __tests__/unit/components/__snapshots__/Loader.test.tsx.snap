// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders a loader component 1`] = `
<div>
  <div
    class="loader p-4 max-w-md w-full"
  >
    <div
      class="animate-pulse flex space-x-4"
    >
      <div
        class="flex-1 space-y-3 py-1"
      >
        <div
          class="h-8 bg-primary-200/20 rounded"
        />
        <div
          class="space-y-3"
        >
          <div
            class="h-2 bg-primary-200/20 rounded w-16"
          />
          <div
            class="grid grid-cols-3 gap-4"
          >
            <div
              class="h-2 bg-primary-200/20 rounded col-span-1"
            />
            <div
              class="h-2 bg-primary-200/20 rounded col-span-2"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
